client/src/views/node.tsx

```diff
@@ -52,15 +52,20 @@ export default class NodeView extends Base<Props, State> {
         const {item, pods, metrics, podMetrics, podsSort} = this.state;
 
         const filteredPods = pods && pods.filter(x => x.spec.nodeName === name);
-        const filteredPodMetrics = getMetrics(filteredPods, podMetrics);
+
+        let filteredPodMetrics = undefined
+        
+        if (item?.metadata) {
+            filteredPodMetrics = getMetrics(filteredPods, podMetrics);
```

Comment:
Maybe we can move the condition check into the `getMetrics` so that it can be applied in multiple places.